applications/views.py

```diff
@@ -98,9 +99,16 @@ def applications_csv(request, city):
     response = HttpResponse(content_type='text/csv')
     response['Content-Disposition'] = u'attachment; filename="{}.csv"'.format(city)
     writer = csv.writer(response)
-    writer.writerow(["Application Number", "email", "Application State", "RSVP Status", "Average Score"])
+    csv_header = ["Application Number","Application State", "RSVP Status", "Average Score"]
+    questions = page.form_set.first().question_set.values_list('title', flat=True)
+    # replace commas with semicolons so it won't confuse csv parsers
+    csv_header.extend([striptags(question) for question in questions])
```

Comment:
You could use `map(striptags, questions)` as a shorter alternative here - probably a matter of preference, so it's just a suggestion :wink: